### ISSUE TITLE
clipping feature have a conflict with angular

### ISSUE THREAD

**Issue by Medo-85** [IFCjs/web-ifc-viewer#119]
Hello, it seems that the clipping feature have a conflict with angular, somehow I tried with latest web-ifc-viewer and doesn't work for any projects 
I also tried with the angular example and I could recreate by doing the following in 'ifc.service.ts' file
```
startIfcViewer(container: HTMLElement) {
    if (!container) return this.notFoundError('container');
    this.container = container;
    this.setupIfcScene();
    this.setupInputs();
    
    window.onkeydown = (event: any) => {
      if(event.code === 'KeyP') {
          this.ifcViewer?.clipper.createPlane();
      }
      else if(event.code === 'KeyO') {
          this.ifcViewer?.clipper.deletePlane();
      }
    };
  }
```

The clipper plane get created but not actual clipping happened as in the following image

![image](https://user-images.githubusercontent.com/80624459/165295563-b82778a0-7ed9-455c-8a68-8a50a3023f6a.png)

**Comment by agviegas** [comment]
Try with the latest version (1.0.177)!

**Comment by Medo-85** [comment]
> Try with the latest version (1.0.177)!

Sure @agviegas will try today.

**Comment by eng-luciano-julien** [comment]
@Medo-85 use deleteAllPlanes() instead of deletePlane() to remove the all the clippings.

**Comment by Medo-85** [comment]
> window.onkeydown = (event: any) => {
>       if(event.code === 'KeyP') {
>           this.ifcViewer?.clipper.createPlane();
>       }
>       else if(event.code === 'KeyO') {
>           this.ifcViewer?.clipper.deletePlane();
>       }

You are right @agviegas tried it today and it works with 1.0.177 thank you.

@eng-luciano-julien Sure this one is very handy instead deleting one by one.
